doc/content/getting-started/aws/ecs/updating/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Updating"
+description: ""
+weight: 3
+---
+
+This page describes the steps for updating {{% tts %}} on AWS ECS.
+
+<!--more-->
+
+Most steps involve updating a CloudFormation stack from our templates. Some steps involve some manual work.
+
+## Overview
+
+From AWS' point of view, updating {{% tts %}} can be divided into three parts:
+
+1. CloudFormation templates
+
+2. Docker images
+
+3. Migrations
+
+## Updating CloudFormation stacks
+
+Updating CloudFormation stacks can be done via the AWS console, using the "Update" and then "Replace current template" buttons. All stacks need to be updated in their numbering order. For example, a stack using template `1-1-vpc` needs to be updated before `1-2-bastion`, which needs to be updated before `2-1-db-aurora-master`.
```

Comment:
I think we should recommend using `Changesets` and not directly `Update`.

Reply:
What do you mean? In this way you create a change set you can then inspect. Also, I assume that anyone doing this knows how to use CloudFormation.

Reply:
There's an option to `Create ChangeSet` in the same dropdown. That's what we use internally and that's what we should recommend.

> Also, I assume that anyone doing this knows how to use CloudFormation.

When there are multiple ways to do something, let's officially recommend the safe way.

 If someone wants to directly press update and not wait for change sets,  that's their choice.

Reply:
I've changed this, although I really don't think that's what we should focus on in this documentation.